src/config/rt.rs

```diff
@@ -1,11 +1,24 @@
 use std::path::PathBuf;
 use std::sync::Arc;
 
-use anyhow::{anyhow, Context, Result};
+use anyhow::{anyhow, bail, Context, Result};
 use http_types::Url;
 
 use crate::config::{ConfigOptsBuild, ConfigOptsClean, ConfigOptsProxy, ConfigOptsServe, ConfigOptsWatch};
 
+/// Config options for the cargo build command
+#[derive(Clone, Debug)]
+pub enum Features {
```

Comment:
Is this approach a good fit for the Project?

Reply:
This seems pretty reasonable.